core/templates/components/cta/cta.twig

```diff
@@ -1,5 +1,31 @@
+{#
+/**
+ * @file
+ * CTA Component.
+ *
+ * Call to action component with clickable image and button.
+ * Variants:
+ *   - Default - Button at the bottom: With full-width button under the image
```

Comment:
I'm open for discussion on this. Do we really need to call out the default variant?

Reply:
I removed the "Default" part. I'm not sure if we need a default or if the button-bottom one should be the default, though I did code it that way (the default class without modifier class is the one with the button at the bottom).